data/bootstrap_seed.yaml

```diff
@@ -0,0 +1,171 @@
+---
+webvirt: 192.168.1.1
+site_name: sitename
+maas_api_key: consumerkey:tokenkey:tokensecret
```

Comment:
Would this be necessary if we can assume the tool we're leveraging to drive this YaML already understands how to talk with MaaS? To me this is configuration passed to drydock when its launched  in a container.